Cargo.lock

```diff
@@ -6019,7 +6098,7 @@ dependencies = [
  "rowan",
  "semver",
  "smallvec",
- "toml",
+ "toml 0.5.11",
```

Comment:
might be worth opening an issue to bump `toml` to `0.7` anywhere else it's used in the sway repo.

Reply:
We would need to upgrade `mdbook`, `proc-macro-crate`, `taplo` and `fuel-etk-ops`. Can this update be addressed in a different PR?

Reply:
Yeah let's do that in a follow up PR. Thanks